Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.1
+FROM ruby:2.6.3
```

Comment:
Here I can recommend the same about Ruby version, it's worth it to use the latest patch version.